### PR TITLE
Validation: Added Unique Descriptions

### PR DESCRIPTION
**Pull request by JustPies**
Fixed:

- description-missing errors (description metadata missing in articles)
- image-alt-text-duplicated

This PR fixes 32 issues across 28 files and is part of ongoing Validation cleanup efforts. No other content has been edited or added.

### CODE REVIEW

File: dotnet-desktop-guide/framework/winforms/controls/how-to-anchor-and-dock-child-controls-in-a-flowlayoutpanel-control.md [hunk 1]
```diff
@@ -39,11 +39,11 @@ The <xref:System.Windows.Forms.FlowLayoutPanel> control supports the <xref:Syste
 
 The following illustration shows four buttons that are anchored and docked relative to the blue button in a <xref:System.Windows.Forms.FlowLayoutPanel>. The <xref:System.Windows.Forms.FlowLayoutPanel.FlowDirection%2A> is <xref:System.Windows.Forms.FlowDirection.LeftToRight>.
 
-![FlowLayoutPanel anchoring](./media/net-flpanchorexp.gif "NET_FLPanchorExp")
+![Illustration of horizontal FlowLayoutPanel, naming four buttons that are anchored and docked to centered button.](./media/net-flpanchorexp.gif "NET_FLPanchorExp")
 
 The following illustration shows four buttons that are anchored and docked relative to the blue button in a <xref:System.Windows.Forms.FlowLayoutPanel>. The <xref:System.Windows.Forms.FlowLayoutPanel.FlowDirection%2A> is <xref:System.Windows.Forms.FlowDirection.TopDown>.
 
-![FlowLayoutPanel anchoring](./media/vs-flpanchor2.gif "VS_FLPanchor2")
+![Illustration of vertical FlowLayoutPanel, naming four buttons that are anchored and docked to centered button.](./media/vs-flpanchor2.gif "VS_FLPanchor2")
```
Comment (MeeraDi) [inline dotnet-desktop-guide/framework/winforms/controls/how-to-anchor-and-dock-child-controls-in-a-flowlayoutpanel-control.md:46]: screenshot

File: dotnet-desktop-guide/framework/winforms/controls/how-to-align-and-stretch-a-control-in-a-tablelayoutpanel-control.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ helpviewer_keywords:
   - "controls [Windows Forms], stretching"
   - "controls [Windows Forms], aligning"
 ms.assetid: 7dc1a157-6fee-4995-8ebc-b65bdc0909a8
+description: Learn how to align and stretch a control in a TableLayoutPanel Control with Anchor and Doc properties.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/controls/how-to-align-and-stretch-a-control-in-a-tablelayoutpanel-control.md:11]:
```suggestion
description: Learn how to align and stretch a control in a TableLayoutPanel control with Anchor and Dock properties.
```

File: dotnet-desktop-guide/framework/winforms/controls/how-to-apply-attributes-in-windows-forms-controls.md [hunk 3]
```diff
@@ -9,6 +9,7 @@ helpviewer_keywords:
   - "attributes [Windows Forms], applying"
   - "Windows Forms controls, applying attributes"
 ms.assetid: af0a3f7f-155b-4ba1-83c4-9cf721331a06
+description: Learn how to apply attributes in Windows Forms controls to develop components and controls that interact with the design environment and execute at run time.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/controls/how-to-apply-attributes-in-windows-forms-controls.md:12]:
```suggestion
description: Learn how to apply attributes in Windows Forms controls to develop components and controls that interact with the design environment.
```

**Review by adegeo — APPROVED** [review]

**Review by adegeo — APPROVED** [review]

File: dotnet-desktop-guide/framework/winforms/controls/how-to-change-styles-on-an-element-in-the-managed-html-document-object-model.md [hunk 4]
```diff
@@ -7,6 +7,7 @@ dev_langs:
 helpviewer_keywords:
   - "managed HTML DOM [Windows Forms], changing styles on elements"
 ms.assetid: 154e8d9f-3e2d-4e8b-a6f3-c85a070e9cc1
+description: Learn how to change styles on an element in the Managed HTML document object model to control the appearance of a document and its elements.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/controls/how-to-change-styles-on-an-element-in-the-managed-html-document-object-model.md:10]:
```suggestion
description: Learn how to change styles on an element in the managed HTML document object model to control the appearance of a document and its elements.
```

**Pull request merged**